projects/ngrx.io/content/guide/router-store/selectors.md

```diff
@@ -0,0 +1,41 @@
+## Router selectors
+
+The `getSelectors` method supplied within `@ngrx/router-store` provides functions for selecting information from router state.
+
+The `getSelectors` method takes a selector function as its only argument to select the piece of state.
+
+Usage:
+
+`reducers/index.ts`
+
+```ts
+import { getSelectors, RouterReducerState } from '@ngrx/router-store';
+
+export interface State {
+  router: fromRouter.RouterReducerState<any>;
+}
+
+export const selectRouter = createFeatureSelector<
+  State,
+  fromRouter.RouterReducerState<any>
+>('router');
+
+const {
+  selectQueryParams,
+  selectRouteParams,
+  selectRouteData,
+  selectUrl,
+} = getSelectors(selectRouter);
+
+// select the current url
+export const selectCurrentUrl = selectUrl;
```

Comment:
These seem to be superfluous. Perhaps redact these from the docs and simply:

```ts
const {
  selectQueryParams,     // select the current route query params
  selectRouteParams,     // select the current route params
  selectRouteData,         // select the current route data
  selectUrl,                      // select the current url
} = getSelectors(selectRouter);
```

Reply:
This is the current style (it's also used in https://ngrx.io/guide/entity/adapter#entity-selectors).
We modified it because the syntax wasn't clear - #1161

I'm open for this change tho 🙂

Reply:
Thanks @ThomasBurleson 
I'll update the docs and push a new commit